pkg/access/bigquery.go

```diff
@@ -88,6 +88,21 @@ func (b Bigquery) AddToAuthorizedViews(ctx context.Context, projectID, dataset,
 		return fmt.Errorf("bigquery.NewClient: %v", err)
 	}
 	defer bqClient.Close()
+	ds := bqClient.Dataset(dataset)
```

Comment:
What I am not sure about is that previously we add an entry to the bigquery metadata everytime we create a dataset. This is accepted by gcp if the metadata belongs to a table, but rejected if it blongs to a view.
Is the duplicated entry intended? If it is not, I think this patch also fixes the duplication entry for tables bug